website/README.md

```diff
@@ -45,7 +45,7 @@ The website can be run locally through node.js or [Docker](https://www.docker.co
 
 Running the site locally is simple. Provided you have Docker installed, clone this repo, run `make`, and then visit `http://localhost:3000`.
 
-The Docker image is pre-built with all the website dependencies installed, which is what makes it so quick and simple, but also means if you need to change dependencies and test the changes within Docker, you'll need a new image. If this is something you need to do, you can run `make build-image` to generate a local Docker image with updated dependencies, then `make website-local` to use that image and preview.
```

Comment:
This is super nit but we spell Docker with an upper case D several other times in this paragraph, I feel like we should change all of them to either Docker or docker

Reply:
Ahh I thought I had reverted this change, good catch thanks! Will update.